lib/platform/github/index.spec.ts

```diff
@@ -108,6 +108,22 @@ describe('platform/github/index', () => {
       ).toMatchSnapshot();
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('should support setup special email for platformCommit', async () => {
+      httpMock.scope(githubApiHost).get('/user').reply(200, {
+        id: 1234,
+        login: 'renovate-bot',
+        name: 'Renovate Bot',
+      });
+      expect(
+        await github.initPlatform({
+          token: '123test',
+          platformCommit: true,
+        } as any)
+      ).toMatchObject({
+        gitAuthor: 'renovate-bot <1234+renovate-bot@users.noreply.github.com>',
```

Comment:
Make local git commit author match its GitHub counterpart, so Renovate won't be bothered by mismatch